tests/host_modules/systemd_service_test.py

```diff
@@ -4,6 +4,9 @@
 from unittest import mock
 from host_modules import systemd_service
 
+REBOOTMETHOD_COLD = 1
```

Comment:
You may not duplicate any logic in code under test.

Reply:
@qiluo-msft  This is not clear?

Reply:
assume you change the named constant value in code, the test code is conflicting with implementation code. You should have single source of truth.

Reply:
Thank you @qiluo-msft , modified the code accordingly.